.github/workflows/ci.yml

```diff
@@ -42,8 +43,9 @@ jobs:
     strategy:
       matrix:
         ruby_version:
-          - "2.7"
+          - "3.3"
           - "3.2"
+          - "3.1"
         include:
           - ruby-version: '2.7'
```

Comment:
Same here as for the unit tests

Reply:
mhm given that this all drops ruby 2.7, I removed the whole block.